homework-submission/b-jason-week1hw/src/ToDoList.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export default class ToDoList extends React.Component {
+    render() {
+        let { todoTask, taskDate } = this.props;
```

Comment:
use const always _(here and every where)_ unless you will change the value for some reason 
And Look Line [34](https://github.com/thwjason/React/blob/jason-3-weekHW/.eslintrc#L34)

__this is not just for following the rules it's for memory usage too, And you have to use on that.__